compute_endpoint/globus_compute_endpoint/endpoint/endpoint.py

```diff
@@ -725,3 +735,16 @@ def get_metadata(config: Config) -> dict:
             log.debug("Config serialization exception details", exc_info=e)
 
         return metadata
+
+    @staticmethod
+    def _run_command(name: str, command: str):
+        log.info(f"running {name} command: {command}")
+        completed_process = subprocess.run(command, shell=True, capture_output=True)
+
+        log.info(f"{name} stdout: {str(completed_process.stdout)}")
+        log.info(f"{name} stderr: {str(completed_process.stderr)}")
```

Comment:
Complete and total bikeshed:

It may be difficult to discern the "sub" log output, given our wrappings and "drab grey text."  If you were feeling super nice, you might consider indenting the sub-log lines.  Something like:
```python
log.info("%s stdout:\n  %s", name, str(completed_process.stdout).replace("\n", "\n  "))
log.info("%s stderr:\n  %s", name, str(completed_process.stderr).replace("\n", "\n  "))
```

Android logs, for example, do a variant of this, where multi-line log records are prefix with a string like `" |  "`.